### PR TITLE
Elmo cu 86874tz0g schedule command to track meetings depending on roles

### PR DESCRIPTION
**Pull request by nlchung**

### CODE REVIEW

**Comment by sherwinchiu** [comment]
Task linked: [CU-86874tz0g /schedule command to track meetings depending on roles](https://app.clickup.com/t/86874tz0g)

File: commands/utility/schedule.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      });
+
+      // output list of events
+      await interaction.reply({ content: eventList, ephemeral: true });
+    }
+	},
+};
+
+async function getRoles(user) {
+  let rolesList = [
```
Comment (sherwinchiu) [inline commands/utility/schedule.js]: is this manually created? what if we add roles in the future (we prob will)

File: commands/utility/schedule.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      timeMin: new Date().toISOString(),
+      maxResults: 15,
+      singleEvents: true,
+      orderBy: 'startTime',
+    });
+    const allEvents = res.data.items;
+
+    let events = [];
+
+    // sort through events and only get add the ones that are applicable by role
```
Comment (sherwinchiu) [inline commands/utility/schedule.js:43]: wait how are events determined? if the string of the role is in the calendar string?

File: commands/utility/schedule.js (hunk 1 above)
Comment (sherwinchiu) [reply commands/utility/schedule.js]: we might want to grab the role list from discord server on deploy commands

File: commands/utility/schedule.js (hunk 2 above)
Comment (nlchung) [reply commands/utility/schedule.js:43]: to be solved; sherwin thinking/discussing with management team(?)
currently determined by roles being in calendar event description


- can intake multiple roles (i.e. user must have both CF Skateboard and electrical roles to be in the CF Skateboard Electrical Meeting)
- have not set it up to handle different non-overlapping roles together, can do so if you want

Comment (sherwinchiu) [reply commands/utility/schedule.js:43]: sg lets just go with your logic for now and if we needa change it we can

**Review by sherwinchiu — APPROVED** [review]
woohoo lets get it in

**Pull request merged**